types/src/v0/v0_1/header.rs

```diff
@@ -13,18 +15,18 @@ use serde::{
 #[derive(Clone, Debug, Deserialize, Serialize, Hash, PartialEq, Eq)]
 pub struct Header {
     /// A commitment to a ChainConfig or a full ChainConfig.
-    pub(crate) chain_config: ResolvableChainConfig,
-    pub(crate) height: u64,
-    pub(crate) timestamp: u64,
-    pub(crate) l1_head: u64,
-    pub(crate) l1_finalized: Option<L1BlockInfo>,
-    pub(crate) payload_commitment: VidCommitment,
-    pub(crate) builder_commitment: BuilderCommitment,
-    pub(crate) ns_table: NsTable,
-    pub(crate) block_merkle_tree_root: BlockMerkleCommitment,
-    pub(crate) fee_merkle_tree_root: FeeMerkleCommitment,
-    pub(crate) fee_info: FeeInfo,
-    pub(crate) builder_signature: Option<BuilderSignature>,
+    pub chain_config: ResolvableChainConfig,
```

Comment:
Mm why do these need to be public?

Reply:
We are using it in reference tests. I think we should move reference tests inside the types crate